<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankWindow</class>
 <widget class="QMainWindow" name="BankWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>741</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>741</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreditCalc by aqualadt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="annuity">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Аннуитетный</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="differ">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дифференцированный</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="SUM">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Общая сумма</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Month">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid gray;</string>
    </property>
    <property name="placeholderText">
     <string>Срок</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percents">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid gray;</string>
    </property>
    <property name="placeholderText">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Срок кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип платежей</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ежемесячный платеж</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Переплата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Общая выплата</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MonthlyPayment">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Overprice">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid gray;</string>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TotalPayment">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid gray;</string>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Result">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans Caption</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
	background-color: rgb(139, 197, 217)
}
QPushButton:pressed {
    background-color: rgb(225, 221, 128)
}
</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>321</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
